README.md

```diff
@@ -222,6 +241,8 @@ The complete list of plugins (i.e. including dependencies) needs to be provided
 Although this document will refrain on detailing the precise details, once the build on the internal Jenkins server is complete,
 the processes will be set in motion to build the `jenkins-2-plugins` RPM that is installed by the [RHEL7 Dockerfile](2/Dockerfile.rhel7) when the next version of the RHEL7 based OpenShift Jenkins image is built.  When new versions of OpenShift are released, associated versions of the RHEL7 based versions of the images produced by this repository are pushed to the Docker registry provided to RHEL7 subscribers.
 
+This plugin RPM is not for public consumption.  If a customer with a subscription wants to get a plugin update, they have to get an update to the OpenShift Jenkins image from the Red Hat Container Catalog.  They do not get access to a new version of the RPM and install it themselves via `yum` or `dnf`.
```

Comment:
i'd leave this out, it belongs in the kbase article(that i guess is still pending) but it seems weird to be in our public/community readme.

Reply:
Yeah this stemmed from lack of optimism that the kbase article won't happen any time soon

I don't know if it is weird to have this in the README, but don't feel strongly about it.

I can remove it.